<!--  This file consumes the information published by detector.launch in bayes_people_tracker to track objects -->
<launch>
    <arg name="param_file" default="$(find rasberry_perception)/src/rasberry_perception/tracking/config/default.yaml" />
    <arg name="target_frame" default="/sequence_colour_frame" />
    <arg name="reset_on" default="" />  <!-- Reset when this topic is published too (std_msgs/String) -->
    <arg name="fps" default="10" />

    <group ns="rasberry_perception">
            <node name="tracking" pkg="rasberry_perception" type="rasberry_perception_tracking" output="screen" respawn="true">
                <rosparam command="load" file="$(arg param_file)"/>
                <param name="target_frame" value="$(arg target_frame)" type="string"/>
                <param name="tracker_frequency" value="$(arg fps)" type="int"/>
                <param name="reset_on" value="$(arg reset_on)" type="string"/>
            </node>
    </group>
</launch>